### PR TITLE
Docs: rework example

### PR DESCRIPTION
**Pull request by lafrech**
- schema registration is optional
- what's the need for "test_request_context"?

### CODE REVIEW

**Comment by sloria** [comment]
Does this work? I thought the app context was necessary for the flask plugin’s path helper.

**Comment by lafrech** [comment]
I just checked and you're right.

Reworked to just remove the useless schema registrations.

**Review by sloria — APPROVED** [review]
The same change should also be made in the README. Otherwise gtg

File: docs/index.rst [hunk 1]
```diff
@@ -66,8 +66,6 @@ Example Application
 
 
     # Register entities and paths
```
Comment (sloria) [inline docs/index.rst]:
```suggestion
    # Register the path and the entities within it
```

**Pull request merged**